partials/map.php

```diff
@@ -9,52 +9,72 @@
     ?>
   </div>
 
-  <div class="map-block">
-    <div class="map-block-content">
-      <img src="<?php echo get_template_directory_uri() . '/dist/img/test-img-3.jpg'; ?>">
-    </div>
-  </div>
-  <div class="map-block">
-    <div class="map-block-content">
-      <img src="<?php echo get_template_directory_uri() . '/dist/img/test-img-4.jpg'; ?>">
-    </div>
-  </div>
-  <div class="map-block">
-    <div class="map-block-content">
-      <img src="<?php echo get_template_directory_uri() . '/dist/img/test-img-1.jpg'; ?>">
-    </div>
-  </div>
-  <div class="map-block">
-    <div class="map-block-content">
-      <img src="<?php echo get_template_directory_uri() . '/dist/img/test-img-2.jpg'; ?>">
-    </div>
-  </div>
-  <div class="map-block">
-    <div class="map-block-content">
-      <img src="<?php echo get_template_directory_uri() . '/dist/img/test-img-4.jpg'; ?>">
-    </div>
-  </div>
-  <div class="map-block">
-    <div class="map-block-content">
-      <img src="<?php echo get_template_directory_uri() . '/dist/img/test-img-3.jpg'; ?>">
-    </div>
-  </div>
-  <div class="map-block">
-    <div class="map-block-content">
-      <img src="<?php echo get_template_directory_uri() . '/dist/img/test-img-1.jpg'; ?>">
-    </div>
-  </div>
-  <div class="map-block">
-    <div class="map-block-content">
-      <img src="<?php echo get_template_directory_uri() . '/dist/img/test-img-2.jpg'; ?>">
-    </div>
-  </div>
-  <div class="map-block">
-    <div class="map-block-content">
-      <img src="<?php echo get_template_directory_uri() . '/dist/img/test-img-3.jpg'; ?>">
-    </div>
+<?php
+  global $grid_group, $group_length, $used_items;
```

Comment:
No need to be aggressive. You are right about the globals but this was the code smell that made me realize this was going the wrong way.